pages/reference/api/get-assets.en-US.mdx

```diff
@@ -52,6 +52,115 @@ curl --location --request GET 'https://livepeer.studio/api/asset/$ASSET_ID' \
 }
 ```
 
+# Retrieve an Asset with IPFS CID or Arweave URL
+
+Retrieves the details of an `Asset` that has previously been
+uploaded using IPFS or Arweave. Supply the unique `CID` from IPFS or `URL` from Arweave associated with that asset, and Livepeer will return the corresponding `Asset` information.
```

Comment:
I think here is a context where the "upload" word can be very confusing. A user without context reading this could be confused, is it because you "uploaded from" IPFS/Arweave, or because you uploaded from anywhere else but then "uploaded to IPFS" (not the way we use "upload", but makes perfect sense) from our API?

The answer here is actually that it means both 😆 
So I was thinking of being explicit the best way we can (given the confusing "upload" term). WDYT of?:
```suggestion
Retrieves the details of an `Asset` that has been
uploaded from IPFS or Arweave, or stored on IPFS from the Livepeer API. Supply the unique `CID` from IPFS or `URL` from Arweave associated with that asset, and Livepeer will return the corresponding `Asset` information.
```